### PR TITLE
Added tasks 932, 933, 934, 935

### PR DESCRIPTION
**Pull request by ThanhNIT**

### CODE REVIEW

File: src/test/java/g0901_1000/s0932_beautiful_array/SolutionTest.java [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+        int[] result = new Solution().beautifulArray(4);
+        int[] expected = new int[] {2, 1, 4, 3};
+        assertThat(CommonUtils.compareArray(result, expected), equalTo(true));
+    }
+
+    @Test
+    void beautifulArray2() {
+        int[] result = new Solution().beautifulArray(5);
+        int[] expected = new int[] {3, 1, 2, 5, 4};
+        assertThat(CommonUtils.compareArray(result, expected), equalTo(true));
```
Comment (javadev) [inline src/test/java/g0901_1000/s0932_beautiful_array/SolutionTest.java:21]: Do we need it? assertThat/equalTo can compare arrays.

Comment (ThanhNIT) [reply src/test/java/g0901_1000/s0932_beautiful_array/SolutionTest.java:21]: items in these array has not same order

Comment (javadev) [reply src/test/java/g0901_1000/s0932_beautiful_array/SolutionTest.java:21]: Ok, let's use this method.

**Comment by javadev** [comment]
✅

**Pull request merged**